packages/discovery-react-components/src/components/CIDocument/components/CIDocument/__tests__/CIDocument.spec.tsx

```diff
@@ -169,7 +169,7 @@ describe('<CIDocument />', () => {
 
     beforeEach(async () => {
       ({ getByTestId, getByText, findByText, findByTestId, queryByTestId } = render(
-        <CIDocument document={shortContract} overrideDocWidth={400} overrideDocHeight={600} />
+        <CIDocument document={shortContract} overrideDocWidth={400} overrideDocHeight={1200} />
```

Comment:
Upgrading `react-virtualized` from `9.21.1` to `9.21.2` or later makes this test unsuccessful.
```
src/components/CIDocument/components/CIDocument/__tests__/CIDocument.spec.tsx
...
  ● <CIDocument /> › Contract › loads correct document

    Unable to find an element with the text: On 22 December 2008 ART EFFECTS LIMITED and Customer. This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.
```

On 9.21.1, with height `600`, entire the document is rendered. i.e. all virtual scroll items `VirtualScroll-8-0 to VirtualScroll-8-128` are rendered. 

But on 9.21.2, with height of `600`, only virtual scroll items `VirtualScroll-8-33 to VirtualScroll-8-128` are rendered. By increasing the height, we can render all the virtual scroll item.

I compared CIDocument between two versions, but I couldn't see difference. So, I update this test case to fix the CI error.

Reply:
I vaguely remember running into issues when upgrading `react-virtualized`. For that reason, I kept it pegged specifically at `9.21.1` (and not something like `^9.21.1`).